_vendor/github.com/chef/chef-workstation/docs-chef-io/content/workstation/getting_started.md

```diff
@@ -42,261 +50,158 @@ These instructions are intended for macOS and Linux users. On Windows Chef Works
+## Setup Chef Credentials
 
-To move files to the `.chef` directory:
+Chef Workstation installs [Chef products and tools]({{< relref "install_workstation.md" >}}) onto your computer.
 
-1. In a command window, enter each of the following:
-
-    ``` bash
-    cp /path/to/config.rb ~/chef-repo/.chef
-    ```
-
-    and:
+The first time you run the Chef Workstation app, it creates a `.chef` directory in your user directory. The `.chef` directory is where you will store your Chef Workstation configuration and your client keys.
 
-    ``` bash
-    cp /path/to/ORGANIZATION-validator.pem ~/chef-repo/.chef
-    ```
-
-    and:
-
-    ``` bash
-    cp /path/to/USERNAME.pem ~/chef-repo/.chef
-    ```
+If you're setting up Chef **as an administrator**, then you will need to manage users with the [Chef Infra Server CLI](https://docs.chef.io/server/ctl_chef_server/#user-management). When you create a new user, Chef Infra Server creates a client for that user with a user-specific RSA client key, which you then need to share securely with that user.
 
-    where `/path/to/` represents the path to the location in which these
-    three files were placed after they were downloaded.
+If you're setting up Chef Workstation **as a user**, then you will need to setup your unique client certificate that corresponds to a client on the Chef Infra Server that your server administrator creates for you. The client certificate is an RSA private key in the `.pem` format.
 
-2. Verify that the files are in the `.chef` folder.
+Save your client certificate in the `.chef` directory using the same name as the client created by your server administrator.
 
-### Without WebUI
+### Configure Your User Credentials File
 
-Use the following steps to manually set up the Chef repository: On your
-Chef Infra Server, create the `ORGANIZATION-validator.pem` and
-`USER.pem` files with the `chef-server-ctl` command line tool. Then, on
-your workstation create the `config.rb` file with the `knife` tool.
+Your `.chef` directory contains an example client `credentials` file, which you can modify to setup communication with your Chef Infra Server.
 
-#### Create an Organization
+At a minimum, you must update the following settings with the appropriate values:
 
-On the Chef Infra Server machine, create the `ORGANIZATION-validator.pem`
-from the command line using `chef-server-ctl`. Run the following
-command:
+- `client_name`: the client name your server administrator created for you
+- `client_key`: the path to the client key in your `.chef` directory
+- `chef_server_url`: the full URL to your Chef Infra Server including the org
 
-``` bash
-chef-server-ctl org-create ORG_NAME ORG_FULL_NAME -f FILE_NAME
-```
+See the [knife config.rb documentation](/workstation/config_rb/) for more
+details.
 
-where
+<!----Tabs Section--->
+{{< foundation_tabs tabs-id="tabs-panel-container" >}}
+{{< foundation_tab active="true" panel-link="config-rb-example" tab-text="Example config.rb">}}
```

Comment:
I think we still need to avoid pulling in this page. It has some stuff like this which needs to get fixed

Reply:
We're in letting the perfect being the enemy of the good. There's room for improvement, but I don't know when I will be able to get back to it--probably not until May (at the soonest).